primo/data_parser/tests/test_well_data.py

```diff
@@ -197,6 +197,28 @@ def test_well_data(caplog, get_column_names):
         "and get_removed_wells_with_reason properties."
     ) in caplog.text
 
+    new_column = np.ones(len(wd.data))
+    with pytest.raises(AttributeError):
+        wd.add_new_column_ordered(["One entry"], new_column)
+    badly_sized_column = np.ones(len(wd.data) - 1)
+    with pytest.raises(AttributeError):
+        wd.add_new_column_ordered(["ones", "col_of_ones"], badly_sized_column)
+
+
+def test_get_well_column_names(get_column_names):
```

Comment:
This test is not doing anything, so please remove it.

Reply:
More tests is not necessarily better :)